fission-server/config/settings.toml

```diff
@@ -2,18 +2,17 @@
 url = "postgres://postgres:postgres@localhost:5432/fission-server"
 connect_timeout = 3
 
-[monitoring]
-process_collector_interval = 10
-
-[otel]
-exporter_otlp_endpoint = "http://localhost:4317"
+[healthcheck]
+enabled = true
+interval_ms = 5000
+max_retries = 3
 
-[server]
-environment = "local"
-did = "did:web:localhost:3000"
-metrics_port = 4000
-port = 3000
-timeout_ms = 30000
+[ipfs]
+peers = [
+  "/ip4/127.0.0.1/tcp/4001",
+  "/ip4/127.0.0.1/tcp/4002/ws",
+  "/ip4/127.0.0.1/udp/4001/quic-v1/webtransport/certhash/uEiC4_sAMUfcxEJtqIlVWRGlHrTSSYuyk5Ulqfl6CjRiOHw/certhash/uEiANUHX9dRBqphQzZINo5WzkStJ7qevCr_2ZAUzLEbqoFw/p2p/12D3KooWLXH5BFzscChpdjtotAvv94hDR6bVxMKLHBHrSnugCRWs"
```

Comment:
Normally you'd set this using an ENV variable, seeing that this last one will be different for each IPFS node. But keeping this here as an example of what to set it to.